packages/api-db-mongodb/src/db/invoice.ts

```diff
@@ -61,7 +61,8 @@ export class MongoDBInvoiceAdapter implements DBInvoiceAdapter {
           paidAt: input.paidAt,
           canceledAt: input.canceledAt,
           sentReminderAt: input.sentReminderAt,
-          items: input.items
+          items: input.items,
+          manuallySetAsPaidByUserId: input.manuallySetAsPaidByUserId
```

Comment:
Why do we want this as different than paidAt?

Reply:
This is for traceability reasons. So that one knows who set the invoice paid as manually, if so.